### PR TITLE
Document how to set up logging per example-app

### PR DESCRIPTION
**Pull request by mathias**
[Rendered](https://github.com/heroku/plexy/blob/mathias-readme-2/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +102,49 @@ Plexy provides some helper functions for taking metric measurements and outputti
...
+config :logger, :console,
+  format: "$time $metadata $message\n",
+  metadata: [:request_id]
+```
+
+Make sure that the `Plexy.RequestId` plug is included in your Elixir app per the Installation instructions, and you will have the `request_id` in the log metadata.
+
 ### Configuring exception reporting
 
+We recommend [Rollbax](https://github.com/elixir-addicts/rollbax) as it lives in your logging backends pipeline. This means that the params retracted in the logging pipeline by your log transformer. To redact certain keys from your Rollbar reporting, see the next section.
```
Comment (Adovenmuehle) [inline README.md]: `This means that the params retracted in the logging pipeline by your log transformer.` is confusing me a bit.

Comment (mathias) [inline README.md]: Yeah, and we're not worried about logging transformers here anymore as a concept. I'll remove this.

Comment (mathias) [inline README.md]: Fixed in d575361afe12d89812e887ca51731c3534426fce.

**Comment by kennyp** [comment]
Looks Good To Me :+1:

**Comment by Adovenmuehle** [comment]
:shipit:

**Pull request merged**